src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -27,36 +33,66 @@ function LineVis(props: Props): ReactElement {
     curveType = CurveType.LineOnly,
     showGrid = true,
     scaleType = ScaleType.Linear,
+    abscissas,
```

Comment:
You could assign a default value to this argument instead of declaring a new `abscissaValues` constant.